pkg/strategy/controller.go

```diff
@@ -78,6 +78,16 @@ func normalize(ctx context.Context, jaeger *v1.Jaeger) {
 		jaeger.Spec.Storage.Type = v1.JaegerMemoryStorage
 	}
 
+	// remove reserved labels
+	delete(jaeger.Spec.JaegerCommonSpec.Labels, "app.kubernetes.io/instance")
```

Comment:
I don't like silently removing things that users explicitly added, but I don't think we have much of a choice here. Perhaps at least add an event to the span when the item existed?

Reply:
Sure. I've added loggings when removing reserved labels.